lib/JsonpMainTemplatePlugin.js

```diff
@@ -36,7 +36,7 @@ JsonpMainTemplatePlugin.prototype.apply = function(mainTemplate) {
 			"var script = document.createElement('script');",
 			"script.type = 'text/javascript';",
 			"script.charset = 'utf-8';",
-			"script.async = true;",
+			"script.async = false;",
```

Comment:
googled a good article about IE loading issues here:
http://www.guypo.com/ies-premature-execution-problem/
and solution is- set src attribute **after** adding element to the dom